crates/viewer/re_view_time_series/src/view_class.rs

```diff
@@ -730,10 +730,11 @@ fn add_series_to_plot(
 
 fn format_time(time_type: TimeType, time_int: i64, timestamp_format: TimestampFormat) -> String {
```

Comment:
Small remark: consider moving this to `TimeType`? That way we could use it in `paint_time_range_ticks` (`re_time_panel`; see comment above).